scene/resources/material.cpp

```diff
@@ -379,6 +379,12 @@ bool ShaderMaterial::_property_can_revert(const StringName &p_name) const {
 			Variant default_value = RenderingServer::get_singleton()->shader_get_parameter_default(shader->get_rid(), *pr);
 			Variant current_value = get_shader_parameter(*pr);
 			return default_value.get_type() != Variant::NIL && default_value != current_value;
+		} else {
+			if (p_name == "render_priority") {
+				return get_render_priority() != 0;
+			} else if (p_name == "next_pass") {
+				return get_next_pass().is_valid();
```

Comment:
These should always return true. The appearance of arrow is determined by `_property_get_revert()`.

Reply:
Thanks. I had tried that but did not seem to work. I now realize that is because `RID()` was wrong too. I was confirming my own mistake. It seems to work with `Variant()` instead.